### PR TITLE
L A R G E Graffiti YIFF IN HELL AND OTHERS

### PR DESCRIPTION
**Pull request by xxalpha**
3x1 graffiti
96x32

Featuring:
 - a secborg cleansing yet another filthy human
 - yiff in hell face
 - paint

It consumes 3 charges, instead of 1, and you need 3 valid tiles to paint otherwise it fails.

ss 2016-06-11 at 01 01 32
untitled

merge token: https://github.com/tgstation/tgstation/pull/18004

:cl:
rscadd: 3x1 grafitti
/:cl:

### CODE REVIEW

**Comment by Cebasiz** [comment]
those are really fucking pretty

maybe a little out of place but really awesome looking

**Comment by Niknakflak** [comment]
The fact that they are pretty makes them not look like graffiti at all. 
Yiff in Hell is really memey and stupid.
And the others are also kinda memey except for the cave drawing eyes.  They also look so massively out of place.

**Comment by xxalpha** [comment]
Graffiti can be pretty.

**Comment by TechnoAlchemisto** [comment]
That doesn't mean anything except that the sprites are up to quality standards. 

Their place in the game is a whole different thing.

**Comment by xxalpha** [comment]
:dog2: :put_litter_in_its_place: :fire:

**Comment by ike709** [comment]
the feature is good but not some of the sprites

**Comment by coiax** [comment]
might I suggest that in order to make complex graffiti, you need to make stencils out of cardboard, and then somehow combine them with the spraycan to make them?

**Comment by xxalpha** [comment]
I will attempt better sprites, but these stay for now.

I am not a spriter to make these sort of things.

**Comment by Durkel** [comment]
Needs a "Fuck the police"

**Comment by bgobandit** [comment]
gotta agree with the rest, these are really pretty but that makes them less graffiti esque

**Comment by MMMiracles** [comment]
make it so the yiff graffiti is linked to having mutantspecies enabled, like the lizard ones are.

**Comment by JJRcop** [comment]
just for fun maybe you could create a counterpart to "yiff in hell", so that way it can just go back and forth with some people painting the positive graffiti and others painting the negative, and promote player conflict, which leads to roleplay!

**Comment by PKPenguin321** [comment]
>a counterpart to "yiff in hell"

what the fuck would that even be?
"hella yiff"?

**Comment by Supermichael777** [comment]
needs to be more runny and fluid, curently it looks square and not at all like spray paint. its too square on the yiff in hell one and too clean on the others

**Comment by xxalpha** [comment]
>"hella yiff"?

Disgusting, PK.

**Comment by Core0verload** [comment]
too memey

**Comment by Steelpoint** [comment]
The sprites just need to look a bit more jaggered, they look more like official posters not something someone painted in a illegal manner.

**Comment by xxalpha** [comment]
@Steelpoint I already done that, I will be updating the PR soon.

**Comment by xxalpha** [comment]
New pictures.

**Comment by Steelpoint** [comment]
Better but holy shit what is that smudge you can smear on the bottom picture?

**Comment by xxalpha** [comment]
It's just covering the wall in paint.

File: code/game/objects/effects/decals/crayon.dm [hunk 1]
```diff
@@ -12,14 +12,17 @@
 	return
 
 
-/obj/effect/decal/cleanable/crayon/New(location, main = "#FFFFFF", var/type = "rune1", var/e_name = "rune", var/rotation = 0)
+/obj/effect/decal/cleanable/crayon/New(location, main = "#FFFFFF", var/type = "rune1", var/e_name = "rune", var/rotation = 0, icon = null)
```
Comment (phil235) [inline code/game/objects/effects/decals/crayon.dm]: don't name an argument the same name as a variable of the object.

**Comment by SidonCC** [comment]
Would be good without the decade old meme. Has nothing to do with ss13.

**Comment by optimumtact** [comment]
Yiff in hell furfags

**Pull request merged**